cromwell_tools/tests/test_cromwell_tools.py

```diff
@@ -234,6 +236,55 @@ def test_validate_cromwell_label_on_valid_labels_str_object(self):
     def test_validate_cromwell_label_on_valid_labels_bytes_object(self):
         self.assertIsNone(cromwell_tools.validate_cromwell_label(json.dumps(self.valid_labels).encode('utf-8')))
 
+    def test_localize_file_https(self):
+        temporary_directory = tempfile.mkdtemp()
```

Comment:
If it works for `https` here, should we update the docstring of `_localize_file` stating that it supports `https` as well?